prepare

```diff
@@ -89,5 +89,30 @@ if [[ $PREPARE_SCRIPT != "null" ]]; then
   echo "$PREPARE_SCRIPT" | $SSH "$(sshUser)@${VM_IP}" bash
 fi
 
+# If a Schutzfile is present and repositories are defined, write them to the specified repofile
+if [ "$(curl -s -o /dev/null -w '%{http_code}' "$CUSTOM_ENV_CI_PROJECT_URL/-/raw/$CUSTOM_ENV_CI_COMMIT_SHA/Schutzfile")" = "200" ]; then
+    SCHUTZFILE=$(curl -L "$CUSTOM_ENV_CI_PROJECT_URL/-/raw/$CUSTOM_ENV_CI_COMMIT_SHA/Schutzfile")
+    KEYS=$(echo "$SCHUTZFILE" | jq keys - | jq -r .[] -)
+    for KEY in $KEYS; do
+        # if a runner's basename starts with the key in a schutzfile,
+        # it's assumed schutzfile[key] applies to that runner
+        if [[ "$(basename "$CUSTOM_ENV_RUNNER")" =~ "$KEY".* ]]; then
+            REPOFILE=
```

Comment:
Did you forget to put something in the var?

Reply:
It gets appended with the repos

Reply:
Oh ok, looked suspicious to me :sweat_smile: it appears that if this line was removed it would still work, but I suppose it can't hurt at least all the vars are together.